packages/design-system-react/src/components/button-base/ButtonBase.stories.tsx

```diff
@@ -90,6 +90,14 @@ const meta: Meta<typeof ButtonBase> = {
       description:
         'Optional prop to pass additional properties to the loading icon',
     },
+    textProps: {
+      control: 'object',
+      description:
+        'Optional props to be passed to the Text component when children is a string',
+      table: {
+        type: { summary: 'Partial<TextProps>' },
+      },
+    },
```

Comment:
Adding textProps to storybook controls

<img width="1506" alt="Screenshot 2025-01-07 at 10 40 27 AM" src="https://github.com/user-attachments/assets/4d249d36-975e-426e-920e-195459cb237c" />